code/modules/client/preferences.dm

```diff
@@ -100,6 +100,7 @@ var/const/MAX_SAVE_SLOTS = 10
 	var/predator_flavor_text = ""
 	//CO-specific preferences
 	var/commander_sidearm = "Mateba"
+	var/co_career_path = "infantry"
```

Comment:
code doc

Reply:
Could you clarify?

Reply:
```js
/// This is a code doc comment! Mind the three slashes.
var/your_var = value

Reply:
No 3 slashes from what I see I added, but I deleted the existing ones I saw. If that makes you happy.